.gitignore

```diff
@@ -12,3 +12,4 @@ doc
 *~
 .*.sw?
 doc-tmp
+.idea
```

Comment:
I don't have a strong opinion about this being added here. However, you can also add it to a global .gitignore file in your home folder. Then you don't need it on a per-project basis.

Reply:
You're right :) fixed